yt/frontends/boxlib/tests/test_outputs.py

```diff
@@ -315,11 +322,28 @@ def test_CastroDataset_2():
     assert isinstance(data_dir_load("castro_sod_x_plt00036"), CastroDataset)
 
 
-@requires_file(LyA)
+@requires_file(plasma)
```

Comment:
why ?

Reply:
If you look inside the test, it's using the `plasma` file and not `LyA`.

Reply:
oh wow how could I miss that. Good catch.